rewrite table

genofire opened the pull request:
Fix scrolling and update just changed

genofire committed:
rework table

genofire committed:
add sorting by abstract table class

genofire committed:
fix editing and update by websocket at same time

oliver commented:
Hab mir den Code selbst nicht genau angesehen, aber von der Benutzung her gefällt mir das gut! Die Suchleiste ist echt gut.

Kleines Problem: beim Sortieren werden nur die gerade angezeigten Knoten sortiert. Wenn man also nur 5 Knoten anzeigen lässt (über die Buttons ganz unten), werden nur diese 5 Knoten umsortiert. Wenn ich also z.B. nur die 5 Knoten sehen will, die sich als letztes gemeldet haben (Lastseen), geht das gar nicht.

oliver committed:
list: fix sorting by Lastseen

lastseen contains a string, so subtraction of the raw value gives
unexpected results.

genofire commented:
Oh ja stimmt, okay paging wird dann auch noch in `Table` ausgelagert ;)

genofire committed:
move maxRow to Table

genofire commented:
@oliver magst du es nochmal testen? (hab gerade nicht die "nötige" Anzahl an routern in meiner DB)

Ansonsten würde ich es morgen mergen
